### PR TITLE
T42016

### PR DESCRIPTION
**Pull request by turrs**
already to review new feature

### CODE REVIEW

File: src/Components/LabelSection/index.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { Badge } from 'antd'
+import './label.css'
+import PropTypes from 'prop-types'
+function LabelSection ({ dataBadge, style, label }) {
```
Comment (argorfc) [inline src/Components/LabelSection/index.jsx]: Please change with arrow function.
*based on pattern

File: src/Components/Link/index.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+import PropTypes from 'prop-types'
+function Link ({ title }) {
```
Comment (argorfc) [inline src/Components/Link/index.jsx]: Please change with arrow function.
*based on pattern

File: src/Parts/AllTrainingEventTable/index.jsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import LabelSection from '../../Components/LabelSection'
+import TableData from '../../Components/TableData'
+import Data from '../../Dummy/DataTable.json'
+import { Rate } from 'antd'
+function AllTrainingEventTable () {
```
Comment (argorfc) [inline src/Parts/AllTrainingEventTable/index.jsx]: Please change with arrow function.
*based on pattern

**Review by argorfc — CHANGES_REQUESTED** [review]
Please read comments

File: src/Components/LabelSection/index.jsx (hunk 1 above)
Comment (turrs) [reply src/Components/LabelSection/index.jsx]: thankyou for your review, this issue fix in 327bdc89e17135a0db052aba8aa4120c69ceb0f2

File: src/Components/Link/index.jsx (hunk 2 above)
Comment (turrs) [reply src/Components/Link/index.jsx]: thankyou for your review, this issue fix in 327bdc89e17135a0db052aba8aa4120c69ceb0f2

File: src/Parts/AllTrainingEventTable/index.jsx (hunk 3 above)
Comment (turrs) [reply src/Parts/AllTrainingEventTable/index.jsx]: thankyou for your review, this issue fix in 327bdc89e17135a0db052aba8aa4120c69ceb0f2

**Comment by turrs** [comment]
> Please read comments

thanyou

**Pull request merged**